examples/hoomd-blue/unbiased/.gitignore

```diff
@@ -0,0 +1 @@
+start.gsd
```

Comment:
Isn't this covered by our root `.gitignore`?

Reply:
I think it is.
I copied it from harmonic_bias example.
Does it hurt?

Reply:
It doesn't, it's just my picky side speaking. I can ignore it for now.